### PR TITLE
add posthog to system

### PR DESCRIPTION
**Pull request by chuyouchia**
Fairly simple, non-intrusive counting analytics using posthog.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -5,6 +5,11 @@ import "react-toastify/dist/ReactToastify.css";
 import FileEditorContainer from "./components/FileEditorContainer";
 import IndexingProgress from "./components/IndexingProgress";
 import InitialSetupSinglePage from "./components/Settings/InitialSettingsSinglePage";
+import posthog from "posthog-js";
+
+posthog.init("phc_xi4hFToX1cZU657yzge1VW0XImaaRzuvnFUdbAKI8fu", {
+  api_host: "https://us.i.posthog.com",
```
Comment (sebastian-philipp) [inline src/App.tsx:11]: a bit :-1: on this. I use ollama and a local tool to avoid sending anything to the internet.

Comment (sebastian-philipp) [reply src/App.tsx:11]: I hear you. Essentially it's your call.

Comment (sebastian-philipp) [reply src/App.tsx:11]: opt-out maybe?

Comment (chuyouchia) [reply src/App.tsx:11]: Yep thanks for the feedback @sebastian-philipp. We will implement opt-out too.

**Review by samlhuillier — APPROVED** [review]
Looks good

**Pull request merged**